### DISCUSSION TITLE
Lellki WP33-EU powerstrip control

### DISCUSSION THREAD

**Discussion by enzopanzani** [Koenkk/zigbee2mqtt#12734] [Devices]
Hello,
I have a Lellki WP33-EU power strip which is well recognized under Z2M. However, it is impossible to control each outlet independently of each other. When you click on a socket, it turns off or on all the others.
Do you know how I can fix this problem?
Thanks a lot for your help !

**Comment by Koenkk** [comment]
Fixed

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by enzopanzani** [reply]
Thank you very much! 
I will try to switch to the dev branch. I'll keep you informed.

**Reply by enzopanzani** [reply]
@Koenkk Everything works ! Many thanks !
